src/params.rs

```diff
@@ -198,7 +198,7 @@ impl Params for &[(&str, &dyn ToSql)] {
 
 macro_rules! impl_for_array_ref {
     ($($N:literal)+) => {$(
-        // These are already generic, and theres a shitload of them, so lets
+        // These are already generic, and there's a shedload of them, so lets
```

Comment:
Yes, shedload is a word in the Cambridge English Dictionary: https://dictionary.cambridge.org/dictionary/english/shedload
And it's nicer than the previously used one, so I see this as an absolute win. :)